cmd/smee/flag.go

```diff
@@ -143,3 +146,59 @@ func newCLI(cfg *config, fs *flag.FlagSet) *ffcli.Command {
 		UsageFunc:  customUsageFunc,
 	}
 }
+
```

Comment:
Not sure why these 3 functions were added, they are not being used anywhere else in the PR

Reply:
yeah, i moved them in from another file that didn't need to exist. Sorry, it was just a clean up item!

Reply:
sounds good, pr LGTM, once merged ill work on moving from boots 0.8.0 to the latest smee version then!